added titles to the docs pages (#109)
dappforce/subsocial-docs: docs/develop/sdk/posts/FetchPosts.md

title: Fetch Posts
displayed_sidebar: developSidebar
---

<head>

>   <title>Fetch Posts on the Subsocial Blockchain | JS SDK Guide</title>

Reviewer: ```suggestion
  <title>How To Get Posts With The Subsocial JS SDK</title>
```
